### PR TITLE
New controller

### PR DESCRIPTION
**Pull request by shella12**
For each URL in the `img` created:

- [ ] A route.

- [ ] An action in the correct controller.

- [ ] A view file.

View files include only placeholders written in plain HTML, e.g:

- Here is a list of posts for a given user

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @posts = Post.all
+  end
 
   def show
     # @postId = Post.find(params[:id])
```
Comment (t-yanick) [inline app/controllers/posts_controller.rb]: - [ ] Great job working on this milestone. Kindly remove any unused or commented pieces of code from your codebase. This makes your codebase clean and profesional.

File: README.md [hunk 2]
```diff
@@ -11,7 +11,6 @@
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
   - [Install](#install)
-  - [Run](#run)
   - [Usage](#usage)
   - [Test](#tests)
 - [👥 Authors](#authors)
```
Comment (t-yanick) [inline README.md:16]: - [ ] Your MIT license link redirects to an error 404 page. It is not a good practice to allow broken links in our README. Kindly fix this.
Screen Shot 2023-03-16 at 11 28 30 PM

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @shella12 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Controllers created without the use of scaffolding
:heavy_check_mark: Routes correctly added

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] As per project requirements, you were supposed to add `views` for each of the `controller actions`. This is currently missing. Kindly add them.
Screen Shot 2023-03-16 at 11 40 11 PM


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
### Hi @shella12, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️ 
- Has users_controller and posts_controller with #index and #show actions. ✔️ 
- Has a corresponding view files for each of the controller actions :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**